### PR TITLE
Leaderboard and Alerts Style Changes

### PR DESCRIPTION
**Pull request by ashtilawat23**
Created template for Alerts page. More style changes for Leaderboard.

### CODE REVIEW

File: src/components/pages/Alerts/RenderAlerts.js [hunk 1]
```diff
@@ -1,11 +1,69 @@
...
+        <div label="Grossman">
+          <div className="under-tabs-container">
+            <div className="flags box"></div>
+            <div className="insights box"></div>
+          </div>
+        </div>
+        <div label="Johnston">
+          <div className="under-tabs-container">
+            <div className="flags box"></div>
+            <div className="insights box"></div>
```
Comment (L-Steinmacher) [inline src/components/pages/Alerts/RenderAlerts.js:33]: Div soup!

**Review by L-Steinmacher — APPROVED** [review]
Looks good!

**Comment by fidan93** [comment]
Looks good! But it is some commented out code. Do we need them?

**Review by fidan93 — APPROVED** [review]

**Review by eet349 — COMMENTED** [review]
Nice job

**Pull request merged**